lib/actions/form.js

```diff
@@ -1,62 +1,41 @@
 import debounce from 'lodash.debounce'
 import { createAction } from 'redux-actions'
 
-import { planTrip } from './api'
+import { planTrip, profileTrip } from './api'
 import { queryIsValid } from '../util/state'
 
-export const settingMode = createAction('SET_MODE')
-export const settingDepart = createAction('SET_DEPART')
-export const settingDate = createAction('SET_DATE')
-export const settingTime = createAction('SET_TIME')
+export const settingQueryParam = createAction('SET_QUERY_PARAM')
 export const changingForm = createAction('FORM_CHANGED')
 
-export function setMode (payload) {
+export function setQueryParam (payload) {
   return function (dispatch, getState) {
-    dispatch(settingMode(payload))
-    dispatch(formChanged())
-  }
-}
-
-export function setDepart (payload) {
```

Comment:
Removing all these is a breaking change.  See my additional discussion on [the commit where this was initially refactored](https://github.com/opentripplanner/otp-react-redux/commit/58530afb5df5c4d69bb4a4aa3906ed84a867b89b#commitcomment-22954224).